app/_locales/en/messages.json

```diff
@@ -1970,7 +1971,8 @@
     "message": "Pending"
   },
   "pendingTransaction": {
-    "message": "You have ($1) pending transaction(s)."
+    "message": "You have ($1) pending transaction(s).",
+    "description": "$1 is count of pending transactions"
```

Comment:
I'm probably the only one who gets really specific about this but I'd really love to have two messages, one singular and one plural.  Since we know the number, we should accommodate this small detail :)

Reply:
Hey @darkwing +1
I have another PR lined up over this, I would prefer to merge it and address the feedback there.

Reply:
I just fixed it